Makefile

```diff
@@ -0,0 +1,5 @@
+build-image:
+	docker build . -t 541497436480.dkr.ecr.eu-west-1.amazonaws.com/protoc
```

Comment:
this the AWS account ID; I'd rather have the entire docker registry URL as a parameter and not hardcode - esp. if it's meant to be sent back as PR? else, this repo should be made private.